debezium-connect-rest-extension/src/test/java/io/debezium/kcrestextension/TestHelper.java

```diff
@@ -48,8 +48,8 @@ public static void setupDebeziumContainer(String debeziumVersion) {
                 .withNetwork(NETWORK)
                 .withCopyFileToContainer(
                         MountableFile.forHostPath(
-                                "target/debezium-connect-rest-extension-1.8.0-SNAPSHOT.jar"),
-                        "/kafka/libs/debezium-kcd-rest-extension-1.8.0.jar")
+                                "target/debezium-connect-rest-extension-1.9.0-SNAPSHOT.jar"),
```

Comment:
Can't we use `Module.version()` or something?

Reply:
@gunnarmorling Not sure about that, that's for https://issues.redhat.com/browse/DBZ-4466 to solve.

Reply:
And `Module.version()` is available for connector Maven modules.

Reply:
Ok, thanks for logging the issue, we can handle it separately then.